### DISCUSSION TITLE
frigate containers

### DISCUSSION THREAD

**Discussion by cyon30** [tteck/Proxmox#807] [Ideas]
Hello

is it possible to make a Fridate nvr containers to easy install on proxmox?

https://frigate.video/

**Comment by norbertjoni** [comment]
Does anyone have a good solution for a frigate? I have a coral usb too

**Reply by bam27beer** [reply]
home automation guy ,on youtube ,been meaning to do mine since october , seems to make it look simple

**Reply by btehan** [reply]
https://github.com/tteck/Proxmox/blob/main/misc/frigate-support.sh

There used to be more instructions on how to use that in the "new scripts" discussion post, but IIRC you make a docker LXC then run that script on the proxmox host and put in the LXC number of your new docker LXC.  Then you just follow the normal docker Frigate install instructions.

**Reply by vednolacni** [reply]
is this stil working? can you use google corale too?
